### PR TITLE
docs: address isInspectable option for WebView

### PR DESCRIPTION
**Pull request by KazuCocoa**
Address https://github.com/appium/appium/issues/14178#issuecomment-1569955191 in the documentation

### CODE REVIEW

File: docs/device-preparation.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Automatic configuration availability is limited on iOS, especially for a real de
 
 Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Advanced_
 
+> **Note**
+> Please make sure the target `WKWebViews` and `JSContexts` components have [`isInspectable`](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable) to make them inspectable via WebView context.
+> Please read [Enabling the Inspection of Web Content in Apps](https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/) about the properly.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:32]: property?

File: docs/device-preparation.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ Automatic configuration availability is limited on iOS, especially for a real de
 
 Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Advanced_
 
+> **Note**
+> Please make sure the target `WKWebViews` and `JSContexts` components have [`isInspectable`](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable) to make them inspectable via WebView context.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:31]: I assume it should be set to true?

File: docs/device-preparation.md [hunk 3]
```diff
@@ -27,6 +27,10 @@ Automatic configuration availability is limited on iOS, especially for a real de
 
 Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Advanced_
 
+> **Note**
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:30]: Not sure if this should be a note. The suggestion is valid for all apps containing web views unless you are going to test Safari

Comment (KazuCocoa) [reply docs/device-preparation.md:30]: This is currently for ios 16.4 sdk built only, so currently set as note. hm, maybe can address SDK version as well..?

**Comment by KazuCocoa** [comment]
Moved to another section, as a WebView

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**